core/modules/taxonomy/taxonomy_vocabulary.class.inc

```diff
@@ -28,6 +28,15 @@ class TaxonomyVocabulary {
    */
   public $description;
 
+  /**
+   * Stores the original TaxonomyVocabulary object before changes during update operations.
+   *
+   * Outside of updates, this property is usually NULL.
+   *
+   * @var TaxonomyVocabulary
```

Comment:
Nice, I think this is allowed to have the same property defined in child classes. And this override uses a more-specific declaration. Seems good to me!